### PR TITLE
Slider component

### PR DESCRIPTION
**Pull request by sahiba0915**
Files to check:

1. slider.html
2. slider.css

Live Preview: https://slidercomp.netlify.app/component/slider/slider.html

### CODE REVIEW

File: component/slider/slider.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  --btn-color: #965c49;
+  --box-shadow: #979494;
+  --primary-font: "Poppins", sans-serif;
+  --primary-alert: #80b918;
+  --success-alert: #dee2e6;
+  --info-alert: #4cc9f0;
+  --warning-alert: #d62828;
+  --default-color: #000000;
+  --pink: #e76f51;
+  --border: rgba(0, 0, 0, 0.3);
```
Comment (akash-1318) [inline component/slider/slider.css:21]: use same color format.

**Review by akash-1318 — COMMENTED** [review]
Added a suggestion besides that one thing i'm concern about is why you are creating color variables in every css file you can just create a utility file for the common code.

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks good.

File: component/slider/slider.css (hunk 1 above)
Comment (sahiba0915) [reply component/slider/slider.css:21]: ok will work on that thanks

**Pull request merged**